content/docs/1.2.0/tutorial/get-to-know-iofog.md

```diff
@@ -0,0 +1,117 @@
+# Get To Know ioFog
+
+In this step of the tutorial we'll better familiarize ourselves with the ioFog environment [we just set up](introduction.html) in the previous step.
+
+## Docker Environment
+
+Verify manually that the ioFog stack containers are correctly started. Note that it may take a minute or two for ioFog to start the tutorial microservices. The output should look like the following.
+
+```console
+$ docker ps --filter "name=iofog"
+
+CONTAINER ID IMAGE                      COMMAND                CREATED            STATUS            PORTS                    NAMES
+baefdede1949 iofog/freeboard:latest     "nginx -g 'daemon of…" 14 seconds ago     Up 13 seconds     0.0.0.0:10102->80/tcp    iofog_BrKHZf9PTcT6yjKcrpnRVBcYPFqxqXxb
+ce6d14551a57 iofog/freeboard-api:latest "node /src/index.js"   17 seconds ago     Up 16 seconds     0.0.0.0:10101->80/tcp    iofog_NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3
+43c00c7c9fc0 iofog/sensors:latest       "node /sensors/index"  20 seconds ago     Up 19 seconds                              iofog_PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb
+5df0afa1018a demo_iofog-agent           "/usr/bin/supervisord" About a minute ago Up About a minute 0.0.0.0:8081->22/tcp     iofog-agent
+7146cd77d400 demo_iofog-controller      "sh -c 'iofog-contro…" About a minute ago Up About a minute 0.0.0.0:51121->51121/tcp iofog-controller
+9e64f5e1afd4 demo_iofog-connector       "sh -c 'service iofo…" About a minute ago Up About a minute 0.0.0.0:53321->8080/tcp  iofog-connector
+```
+
+We can also see that in the PORTS column some of these containers have published [port mappings](https://docs.docker.com/config/containers/container-networking/):
+
+For example, there's a microservice running an instance of [Freeboard](https://github.com/Freeboard/freeboard), an open source visualization dashboard for IoT devices. It's running a web server that serves the dashboard. Go ahead and try it out at http://localhost:10102/?load=dashboard.json
+
+The Freeboard microservice doesn't know it's running locally, so it could just as well be running on real edge node hardware!
+
+## Shell Into Our Containers
+
+Our tutorial environment has three ioFog containers:
+
+- iofog-agent (see [Agents](../agents/overview.html))
+- iofog-controller (see [Controller](../controllers/overview.html))
+- iofog-connector (see [Connector](../connectors/overview.html)
+
+We can think of each of these containers as if they were deployed on separate devices. In production, our Controller is most often running on a cloud server and our Agents are each running on individual edge hardware nodes in the field. The Controller is controlling the Agent the same way it would if the devices were hundreds of miles away, and the Connector can broker communication between any microservices we run.
```

Comment:
`In production, our Controller is most often running on a cloud server`

In production? Shouldn't the wording be more like "Controllers are generally running on more conventional networked hardware, i.e. Servers with ethernet connections, or cloud servers?

That's vs. the Agent which in production will almost always be less network/resource intensive hardware.